cpp/benchmarks/CMakeLists.txt

```
@@ -183,6 +183,11 @@ ConfigureNVBench(
   sort/sort_lists.cpp sort/sort_structs.cpp
 )
 
+# ##################################################################################################
+# * create benchmark
+# --------------------------------------------------------------------------------
+ConfigureNVBench(STRUCT_CREATION_NVBENCH create/create_structs.cpp)
```

```suggestion
ConfigureNVBench(STRUCT_CREATION_NVBENCH structs/create_structs.cpp)
```
I'd prefer this to be in a `structs` sub-directory at this time.

---

done!